doc/wave_calib.rst

```diff
@@ -84,6 +85,47 @@ vary (e.g. grating tilts).  We are likely to implement
 this for echelle observations (e.g. HIRES).
 
 
+Identify
+--------
+
+If you would prefer to manually wavelength calibrate, then
+you can do so with the 'identify' task. To launch this task,
+you need to have successfully traced the slit edges (i.e. a
+MasterEdges file must exist), and generated a MasterArc
+calibration frame. To launch a GUI, use the following command:
+
```

Comment:
Add in two colons or use `code-block`, like this:

```
...
calibration frame. To launch a GUI, use the following command:

.. code-block:: bash

     pypeit_identify MasterArc_A_1_01.fits 

 Instructions on how to use this GUI are available by pressing 
...
```

Reply:
Done!